### PR TITLE
👽️ Update erc721 dependency

### PR DESCRIPTION
**Pull request by bal7hazar**

### CODE REVIEW

File: Scarb.toml [hunk 1]
```diff
@@ -6,8 +6,8 @@ version = "2.0.2"
 
 [dependencies]
 starknet = ">=2.2.0"
-openzeppelin = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", tag = "v0.7.0-rc.0" }
-cairo_erc_721 = { git = "https://github.com/carbonable-labs/cairo-erc-721" }
+openzeppelin = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", rev = "fcdd6d1b02306f18f1089656f15b87d1701ffde3" }
+snforge_std = { git = "https://github.com/foundry-rs/starknet-foundry.git", tag = "v0.6.0" }
 
 [[target.starknet-contract]]
 sierre = true
```
Comment (tekkac) [inline Scarb.toml]:
```suggestion
sierra = true
```

File: src/extensions/slotapprovable/interface.cairo [hunk 2]
```diff
@@ -17,7 +17,7 @@ trait IERC3525SlotApprovable<TContractState> {
 }
 
 #[starknet::interface]
-trait IERC3525SlotApprovableLegacy<TContractState> {
+trait IERC3525SlotApprovableCamelOnly<TContractState> {
```
Comment (tekkac) [inline src/extensions/slotapprovable/interface.cairo:20]:
```suggestion
trait IERC3525SlotApprovableKamelOuali<TContractState> {
```

File: src/extensions/slotapprovable/module.cairo [hunk 3]
```diff
@@ -32,6 +38,10 @@ mod ERC3525SlotApprovable {
         approved: bool,
     }
 
+    mod Errors {
```
Comment (tekkac) [inline src/extensions/slotapprovable/module.cairo:41]: Cool idea

**Review by tekkac — DISMISSED** [review]
LGTM :ok_hand:

File: src/extensions/slotapprovable/module.cairo (hunk 3 above)
Comment (bal7hazar) [reply src/extensions/slotapprovable/module.cairo:41]: From OpenZeppelin :sweat_smile:

**Pull request merged**